views/document-upload.html

```diff
@@ -1,10 +1,7 @@
 {% extends "layout.html" %}
 
 {% block pageTitle %}
-  {% if errorList.length > 0 %}
-    Error:
-  {% endif %}
-  Object to a company being struck off
+  Add documents to support your objection
```

Comment:
We still need the Error shown - see the govuk guidelines - https://design-system.service.gov.uk/components/error-summary/

Reply:
Is that just for the two pages I've updated or for all pages?

Reply:
I'm just looking at this PR, I'm assuming the other pages are ok. Looks like it is needed for screen readers when there is a validation error. If there is a Error: condition, i'd leave it in.

Reply:
Put this back - commit should read "OBJ-149PR added error prefix" not "309"